cachelib/allocator/MM2Q.h

```diff
@@ -68,6 +68,7 @@ class MM2Q {
   enum LruType { Warm, WarmTail, Hot, Cold, ColdTail, NumTypes };
 
   // Config class for MM2Q
+  // TODO: implement support for useCombinedLockForIterators
```

Comment:
Given the code you have, implementing this for 2Q is just adding the if statement you have for MMLRU right?

Reply:
Adding the if statement + extending the MM2Q config

Reply:
Done. I just realized that TinyLFU uses a spin lock, which does not support combined locking, so there is no option to enable it.